Mathlib/Topology/Inseparable.lean

```diff
@@ -228,12 +228,205 @@ theorem IsClosed.continuous_piecewise_of_specializes [DecidablePred (· ∈ s)]
     Continuous (s.piecewise f g) := by
   simpa only [piecewise_compl] using hs.isOpen_compl.continuous_piecewise_of_specializes hg hf hspec
 
+attribute [local instance] specializationPreorder
+
 /-- A continuous function is monotone with respect to the specialization preorders on the domain and
 the codomain. -/
-theorem Continuous.specialization_monotone (hf : Continuous f) :
-    @Monotone _ _ (specializationPreorder X) (specializationPreorder Y) f := fun _ _ h => h.map hf
+theorem Continuous.specialization_monotone (hf : Continuous f) : Monotone f :=
+fun _ _ h => h.map hf
```

Comment:
```suggestion
  fun _ _ h => h.map hf
```
and (almost) everywhere below
(I don't think term-mode proof/defs are exempt from indentation ...)

Reply:
I thought tactics were indented because they were in a `by`-block, and terms are not in a block? 
I'll indent them anyway.